doc/FLPsuite.md

```diff
@@ -0,0 +1,208 @@
+ninja -j16 install 
+```
+
+_**Compilation on top of a local O2**_
+
+If you want to build also O2 locally do
+
+```
+# O2
+git clone https://github.com/AliceO2Group/AliceO2.git
+cd AliceO2
+git checkout <release> # use the release included in the installed FLP suite
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=/tmp/installdir -G Ninja -DCLANG_EXECUTABLE=/opt/o2/bin-safe/clang -DCMAKE_BUILD_TYPE=RelWithDebugInfo ..
+ninja -j8 install
+
+# QC
+git clone https://github.com/AliceO2Group/QualityControl.git
+cd QualityControl
+git checkout <release> # use the release included in the installed FLP suite
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=/tmp/installdir -G Ninja -DCLANG_EXECUTABLE=/opt/o2/bin-safe/clang -DCMAKE_BUILD_TYPE=RelWithDebugInfo -DO2_ROOT=/tmp/installdir ..
+ninja -j8 install
+```
+
+_**Important step in case several nodes are involved**_
+
+In case the workflows will span over several FLPs and/or QC machines, one should `scp` the `installdir` to the other machines in the same directory.
+
+**Use it in aliECS**
+
+In the aliECS gui, in the panel "Advanced Configuration", et an extra variable `extra_env_vars` and set it to
+
+```
+PATH=/tmp/installdir/bin/:$PATH; LD_LIBRARY_PATH=/tmp/installdir/lib/:/tmp/installdir/lib64/:$LD_LIBRARY_PATH; QUALITYCONTROL_ROOT=/tmp/installdir/; echo
+```
+
+Replace `/tmp/installdir` with your own path. Make sure that the directory is anyway readable and traversable by users `flp` and `qc`
+
+## Switch detector in the workflow _readout-dataflow_
+
+The workflow readout-dataflow works by default with the detector code TST. To run with another detector (e.g. EMC) do:
+
+2. Replace all instances of `TST` in the QC config file in consul with the one of the detector (e.g. `EMC`).
+2. Set the variable `detector` in aliECS to the detector (e.g. `EMC`).
+
+## Get all the task output to the infologger
+
+Set the variable log_task_output=all
+
+## Using a different config file with the general QC
+
+One can set the `QC URI` to a different config file that is used by the general QC when enabled. However, this is not the recommended way. One must make sure that the name of the task and the check are left untouched and that they are both enabled.
+
+## Enable the repo cleaner
+
+If the CCDB used in an FLP setup is the local one, the repo cleaner might be necessary as to avoid filling up the disk of the machine.
+
+By defaults there is a _disabled_ cron job :
+
+```shell
+*/10 * * * * /opt/o2/bin/o2-qc-repo-cleaner --config /etc/flp.d/ccdb-sql/repocleaner.yaml --dry-run > /dev/null 2>> /tmp/cron-errors.txt
+```
+
+1. copy the config file /etc/flp.d/ccdb-sql/repocleaner.yaml
+2. modify the config file to suit your needs
+3. run by hand the repo-cleaner to check that the config file is ok
+3. update the cron job to use the modified config file
+4. uncomment the cron job
+
+## Definition of new arguments
```

Comment:
i think this chapte rhas nothing to do with FLP suite, it seems like DPL documentation. Personally, I would remove it completely - we (developers) know how to do it or can easily find out, users don't need to do it since we have a global `o2-qc` executable. This was probably useful when users had their own QC execs (<2019 probably).

Reply:
yes, moved to `DevelopersTips.md`